src/SprykerSdk/Zed/AopSdk/Communication/Console/BuildCodeFromOpenApiConsole.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Zed\AopSdk\Communication\Console;
+
+use Generated\Shared\Transfer\OpenApiRequestTransfer;
+use Generated\Shared\Transfer\OpenApiResponseTransfer;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * @method \SprykerSdk\Zed\AopSdk\Business\AopSdkFacadeInterface getFacade()
+ */
+class BuildCodeFromOpenApiConsole extends AbstractConsole
+{
+    /**
+     * @var string
+     */
+    public const OPTION_OPEN_API_FILE = 'openapi-file';
+
+    /**
+     * @var string
+     */
+    public const OPTION_OPEN_API_FILE_SHORT = 'a';
+
+    /**
+     * @var string
+     */
+    public const OPTION_ORGANIZATION = 'organization';
+
+    /**
+     * @var string
+     */
+    public const OPTION_ORGANIZATION_SHORT = 'o';
+
+    /**
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('build:from:openapi')
+            ->setDescription('Builds code from an OpenAPI file definition.')
+            ->addOption(static::OPTION_OPEN_API_FILE, static::OPTION_OPEN_API_FILE_SHORT, InputOption::VALUE_REQUIRED, '', $this->getConfig()->getDefaultOpenApiFile())
+            ->addOption(static::OPTION_ORGANIZATION, static::OPTION_ORGANIZATION_SHORT, InputOption::VALUE_REQUIRED, 'Namespace that should be used for the code builder. When set to Spryker code will be generated in the core modules.', 'App');
```

Comment:
We need an additional option for our different types of API applications. We have BackendApi and StorefrontApi. Depending on the application type we will build different code.

Please add `APPLICATION_TYPE` and add `backend` as default. Later on we will also add code for the frontend one.